src/Functions/date_trunc.cpp

```diff
@@ -20,131 +20,163 @@ namespace ErrorCodes
 namespace
 {
 
-class FunctionDateTrunc : public IFunction
-{
-public:
-    static constexpr auto name = "dateTrunc";
+    class FunctionDateTrunc : public IFunction
```

Comment:
I suppose the automatic style formatter was a bit harsh, consider using `.clang-format` at the top of the repository

Reply:
It seems that running `clang-format -i -style=file src/Functions/date_trunc.cpp` gives me what I originally pushed. I just undid the indentation manually. Let me know if any of the other formatting needs to be redone.